integration/README.md

```diff
@@ -65,17 +65,41 @@ All config options are optional. If you don't pass any config, it will use the d
 | `theme`           | `'light' or 'dark'`                                | The theme of the chat. If omitted, it will use the system preferences or user's last theme used in <https://grill.chat>                                                                                                                                                 |
```

Comment:
```suggestion
| `theme`           | `'light' or 'dark'`                                | The theme of the chat. If omitted, it will use the system preferences or the last <https://grill.chat> theme selected by the user.                                                                                                                                                |
```